docs/components/tasklist/userguide/starting-processes.md

```diff
@@ -44,20 +44,6 @@ There could be multiple reasons why you are not seeing any process in the **Proc
 
 For all the above scenarios, contact your administrator to understand why no processes are displayed.
 
-## Start public processes via form
+## Publishing your process
```

Comment:
> @christinaausley We can completely remove this section that describes that process publishing via a start form is not supported on 8.7. This is already mentioned in the documentation related to process publishing in Web Modeler. That way, this document only describes how to start a process from Tasklist.
Wdyt?

For some reason this comment isn't showing up anymore -- @volodymyr-melnykc do you still feel this way?